src/js/view/view.js

```diff
@@ -186,6 +186,12 @@ function View(_api, _model) {
             _currentlyFloating = floating;
             _this.trigger(FLOAT, { floating });
             updateVisibility();
+
+            // On iOS, an ad may receive the original resize event before wrapper is revealed.
```

Comment:
This is a mobile UI issue, it is not specific to iOS, but to our floating implementation when either `isMobile` is true or the screen size is small.